converter/src/main/java/org/dswarm/converter/flow/TransformationFlow.java

```diff
@@ -221,129 +218,92 @@ public String apply(
 				.setReceiver(gdmModelsTimer)
 				.setReceiver(writer);
 
-		opener.process(tuples);
-		opener.closeStream();
-		// objectReceiver.closeStream();
-
-		// return stringWriter.toString();
-
-		morphContext.stop();
-		final Context gdmContext = gdmTimer.time();
-
-		final ImmutableList<GDMModel> gdmModels = writer.getCollection();
-
-		final Model model = new Model();
-		String recordClassUri = null;
-
-		final Optional<String> optionalDataModelSchemaRecordClassURI = getDataModelSchemaRecordClassURI();
-
-		final String defaultRecordClassURI;
-
-		if (optionalDataModelSchemaRecordClassURI.isPresent()) {
-
-			defaultRecordClassURI = optionalDataModelSchemaRecordClassURI.get();
-		} else {
-
-			defaultRecordClassURI = ClaszUtils.BIBO_DOCUMENT_URI;
-		}
-
 		// transform to FE friendly JSON => or use Model#toJSON() ;)
 
-		final ObjectMapper objectMapper = DMPPersistenceUtil.getJSONObjectMapper();
-		// final ArrayNode result = objectMapper.createArrayNode();
-		final Set<String> recordURIs = Sets.newLinkedHashSet();
-
-		final List<GDMModel> finalGDMModels = Lists.newArrayList();
-
-		for (final GDMModel gdmModel : gdmModels) {
-
-			// result.add(rdfModel.toRawJSON());
+		final Optional<String> optionalDataModelSchemaRecordClassURI =
+				getDataModelSchemaRecordClassURI();
 
-			if (gdmModel.getModel() == null) {
```

Comment:
this, is a point where I'm always wondering (and stopped my review last time ...). I don't really grasp the business logic here, i.e., why do we need to process an empty model. anyway, it working right now ...